docs/rest_api.md

```diff
@@ -2,3 +2,266 @@
+```bash
+curl -u $AUTH -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/actions/hello?overwrite=true -d '{"namespace":"_","name":"hello","exec":{"kind":"nodejs:6","code":"function main(params) { return {payload:\"Hello \"+params.name}}"},"annotations":[{"key":"web-export","value":true},{"key":"raw-http","value":false},{"key":"final","value":true}]}'
+```
+You can now invoke this action as a public url with no OpenWhisk authorization. Try invoking using the web action public URL including an optional extension such as `.json` or `http` for example at the end of the URL.
+```bash
+curl https://openwhisk.ng.bluemix.net/api/v1/web/john@example.com_dev/default/hello.json?name=John
+```
+```json
+{
+  "payload": "Hello John"
+}
+```
+
+## Sequences
+
+To create an action sequence, you need to create it by providing the name of the actions that compose the sequence in the desired order, so the output from the first action is passed as input to the next action.
+
+$ wsk action create sequenceAction --sequence /whisk.system/utils/split,/whisk.system/utils/sort
+
+Create a sequence with the actions `/whisk.system/utils/split` and `/whisk.system/utils/sort`.
+```bash
+curl -u $AUTH -d '{"namespace":"_","name":"sequenceAction","exec":{"kind":"sequence","components":["/whisk.system/utils/split","/whisk.system/utils/sort"]},"annotations":[{"key":"web-export","value":true},{"key":"raw-http","value":false},{"key":"final","value":true}]}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/actions/sequenceAction?overwrite=true 
+```
+
+## Triggers
+
+To create a trigger, the minimum information you need is a name for the trigger, you could also include default parameters that get passed to the action thru a rule when the trigger gets fired.
+
+Create a trigger with name `events` with a default parameter `type` with value `webhook` set.
+```bash
+curl -u $AUTH -d '{"name":"events","parameters":[{"key":"type","value":"webhook"}]}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/events?overwrite=true 
+```
+
+Now whenever you have an event that needs to fire this trigger it just takes a HTTP request with a method `POST` using the OpenWhisk Authorization key.
+
+To fire the trigger `events` with a parameter `temperature` send the following HTTP request.
+
+```bash
+curl -u $AUTH -d '{"temperature":60}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/events
+```
+
+### Triggers with Feed Actions
+
+There are special triggers that can be created using a feed actions, the feed action it's an action that helps with the configuration of a feed provider that will be in charge of firing the trigger whenever there is an event for the trigger. Learn more how these feed providers in the [feeds.md] documentation.
```

Comment:
end sentence after feed action with period. Start new sentence: The feed action is an ...